plugins/talk-plugin-downvote/client/components/Icon.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import cn from 'classnames';
+
+export default ({ className }) => (
+  <i
+    className={cn('fa', 'fa-arrow-circle-down', className)}
```

Comment:
This could possibly be an issue in the future since I don't know if we are planning on keeping font awesome after the styling refactor.

Reply:
Ah yeah, good point!